src/components/NavBar/index.ts

```diff
@@ -0,0 +1 @@
+export * from './NavBar';
```

Comment:
Please avoid exporting directly from an import. Instead, import the module and then export it separately.